Code/Framework/AzNetworking/AzNetworking/ConnectionLayer/ConnectionEnums.h

```diff
@@ -54,6 +54,7 @@ namespace AzNetworking
         , ConnectionRejected
         , ConnectionDeleted
         , ServerNotReady
+        , ServerNoLevelLoaded
```

Comment:
This feels very similar to "ServerNotReady". But I see we just return an error code and not error code + exception/error message, so in our event based view we can't provide a context.

Reply:
Good call out. If Disconnect() also accepted an error message, then we could roll these into a single "ServerNotReady" enum and explain why it's not ready in a message. However, Disconnect() only takes a DisconnectReason and no error. 
Also, ServerNotReady isn't actually used anywhere. Deleted.